spec/lib/given/extensions_spec.rb

```diff
@@ -180,6 +180,16 @@
     Then { result == :ok }
   end
 
+  if RSpec::Version::STRING.start_with? "2"
```

Comment:
Hey @ronen, could you explain this branch for me? Are you meaning to check for RSpec <= 2 or is there something about 2 specifically?